docs/source/tutorials/faq.ipynb

```diff
@@ -0,0 +1,89 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ffe0d62e",
+   "metadata": {},
+   "source": [
+    "# FAQ\n",
+    "\n",
+    "Answers to frequently asked questions about the [cleanlab](https://github.com/cleanlab/cleanlab) open source package"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eca36874",
+   "metadata": {},
+   "source": [
+    "### How do I format labels for Cleanlab?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d5d0fbb3",
+   "metadata": {},
+   "source": [
+    "Cleanlab only works with integer-encoded labels in the range `{0,1, ... K-1}` where `K = number_of_classes`. Final labels array should only contain integer values in the range  `{0, K-1}` and be of shape `(N,)` where `N = total_number_of_data_points`.\n",
+    "\n",
+    "**Text labels** should to be integer-encoded. An example of text to integer-encoding can be found in step 2 of [Text Classification with TensorFlow, Keras, and Cleanlab Tutorial](https://docs.cleanlab.ai/v2.0.0/tutorials/text.html#2.-Load-and-preprocess-the-IMDb-text-dataset). \n",
```

Comment:
First sentence can be improved as:

**Text or string labels** should be mapped to integers for each possible value.

Reply:
Just FYI:  links in the documentation should not be for specific version of cleanlab.  In this case the link should've been: have to work for every single version of the documentation, so there should never be a hardcoded global link to another internal page in the documentation.  In this case, the link should've simply been:  https://docs.cleanlab.ai/stable/tutorials/text.html#2.-Load-and-preprocess-the-IMDb-text-dataset

Reply:
But actually in this case specifically, I would not use a link since the concept is so simple to explain. I would instead just say:

For example if your original data labels look like this: `["dog", "dog", "cat", "mouse", "cat"]`, you should feed them to Cleanlab like this: `labels = [1, 1, 0, 2, 0]` and keep track of which integer represents which class (classes were ordered alphabetically in this example).